catalog-items.html

```diff
@@ -0,0 +1,371 @@
+
+                <li class="nav-item"> 
+                    <a href="index.html" class="nav-dropbtn" >About</a>
+
+                    <ul class="nav-dropdown-content">
+                        <a href="catalog.html" class="nav-link">Lorem.</a>
+                        <a href="contact.html" class="nav-link">Lorem.</a>
+                    </ul>
+                </li>
+            <!-- and so on -->
+            </ul>
+        </nav>
+       
+        <button class="btn-theme-toggle" onclick="toggleTheme()">
+            <img src="./img/switch-sun.svg" alt="sun" width="20" height="20" loading="lazy" class="theme-toggle-img">                   
+        </button>
+       
+
+        <div class="page-header-connect">
+            <a href="https://discord.com" target="_blank" class="link-discord">  
+                <img src="./img/Discord.svg" alt="discord" width="20" height="20" loading="lazy" class="logo-discord">
+                Discord              
+            </a>
+
+            <a href="https://connect-wallet.html" target="_blank" class="link-connect-wallet">
+                <img src="./img/wallet.svg" alt="wallet" width="20" height="20" loading="lazy" class="logo-wallet">
+                Connect
+            </a>                              
+        </div>
+    </header>
+    
+    <main class="content">
+        <section class="item-detail">
+            <header class="item-detail-heading">  <!-- если header уже имеет класс, есть ли смысл прописывать класс h, если кроме него там ничего нет? -->
+                <h1>ITEM DETAIL</h1> 
+            </header>
+           
+            <nav class="item-detail-nav">
+                <ul class="item-detail-navlist hidden">
+                    <li class="item-detail-navitem">
+                        <a href="index.html" class="item-detail-navlink">Home</a>                       
+                    </li>
+
+                    <div class="vertical-line">&VerticalLine;</div> <!-- <hr class="vertical-line"> - этот вариант не очень или на уровне с этим? -->
+                   
+                    <li class="item-detail-navitem">
+                        <a href="catalog-items.html" class="item-detail-navlink">Item detail</a>
+                    </li>                    
+                </ul>
+            </nav>
+        
+            <article class="product-card">
+                 <figure class="product-info"> 
+                    <img src=".//img/item2.svg" alt="" width="400" height="400" loading="lazy" class="product-img">
+
+                    <figcaption class="product-name">SKISIRS#4587</figcaption>
+                </figure>
+
+                <p class="product-text">
+                    As the first hero of the Neoh Universe, collection has over 9,999 unique skins drawn from the different missions and challenges he faced throughout his life.
+                </p>
+
+                <dl class="product-features">
+                    <dt class="product-termin">Clothing
```

Comment:
dt and dd it's a separate elements
please dont' include dd into dt
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl